docs/r2/hyperparameter_tuning_with_hparams.ipynb

```diff
@@ -0,0 +1,599 @@
+            "10000/10000 [==============================] - 0s 41us/sample - loss: 1.3227 - accuracy: 0.6278\n",
+            "--- Running training session 7\n",
+            "{'num_units': 32, 'dropout_rate': 0.2, 'optimizer': 'adam'}\n",
+            "60000/60000 [==============================] - 4s 64us/sample - loss: 0.6546 - accuracy: 0.7748\n",
+            "10000/10000 [==============================] - 0s 39us/sample - loss: 0.4731 - accuracy: 0.8340\n",
+            "--- Running training session 8\n",
+            "{'num_units': 32, 'dropout_rate': 0.2, 'optimizer': 'sgd'}\n",
+            "60000/60000 [==============================] - 3s 55us/sample - loss: 1.6611 - accuracy: 0.4529\n",
+            "10000/10000 [==============================] - 0s 40us/sample - loss: 1.1841 - accuracy: 0.6557\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "metadata": {
+        "id": "qSyjWQ3mPKT9",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## 4. Visualize the results in TensorBoard's HParams plugin"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "5VBvplwyP8Vy",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "The HParams dashboard can now be opened. Start TensorBoard and click on \"HParams\" at the top."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "Xf4KM-U2bbP_",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "%tensorboard --logdir logs/hparam_tuning"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "UTWg9nXnxWWI",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "<img class=\"tfo-display-only-on-site\" src=\"images/hparams_table.png?raw=1\"/>"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "4RPGbR9EWd4o",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "The left pane of the dashboard provides filtering capabilities:\n",
+        "- Filter which hyperparameters/metrics are shown in the dashboard\n",
+        "- Filter which hyperparameter/metrics values are shown in the dashboard\n",
+        "- Filter on run status (running, success, ...)\n",
+        "- Sort by hyperparameter/metric in the table view\n",
+        "- Number of session groups to show (useful for performance when there are many experiments)\n"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "Z3Q5v28XaCt8",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "The HParams dashboard has three different views, with various useful information:\n",
+        "* The **Table View** lists the runs, their hyperparameters, and their metrics.\n",
```

Comment:
Add that you can click on a table row / par-coords line / scatter-plot marker and see a plot of the metrics as a function of training step for that session. I know that in this tutorial we have just one step, so the line won't be very interesting, but we can point to that capability which I think many users will find valuable.

Reply:
Added.